libclc/utils/libclc-remangler/CMakeLists.txt

```diff
@@ -24,3 +24,9 @@ clang_target_link_libraries(libclc-remangler
   clangSerialization
   LLVMOption
   )
+
+# If we've not already imported a libclc-remangler tool from an external build,
+# set it up now.
```

Comment:
Is this a new feature? that we can now use an existing remangler?

Reply:
I don't believe it's a new feature, it should currently be handled by the `LIBCLC_CUSTOM_LLVM_TOOLS_BINARY_DIR` option importing `libclc-remangler` from another location. However since I realise I've never tried that option myself, and I don't know how it works (shouldn't there be a clash on the `libclc-remangler` target?) I've kicked off a build locally and will get back to you once I work out what's going on.

Reply:
Aye unfortunately it seems that there's currently a bug where `LIBCLC_CUSTOM_LLVM_TOOLS_BINARY_DIR` *says* that the `libclc-remangler` it taken from that path, but in fact it's always using the built one. It's because we set the `LIBCLC_REMANGLER` variable, but then use the `libclc-remangler` target to build.

I think we can consider this change a bug fix. I'll update the PR description.